src/hyrdate.js

```diff
@@ -1,20 +1,28 @@
 class Hydration {
-  constructor(hydrationData) {
-    this.userId = hydrationData.userID,
-    this.date = hydrationData.date,
-    this.numOunces = hydrationData.numOunces
+  constructor(hydrationData, userID) {
+    this.userID = userID;
+    this.hydrationData = hydrationData
   }
 
-  calculateAverageDailyConsumption() {
-
+  calculateAverageTotalConsumption() {
+    return Math.floor(this.hydrationData.reduce((acc, cur) => {
+      console.log(cur.numOunces);
+      acc += cur.numOunces;
+      return acc;
+    }, 0 )) / this.hydrationData.length
   }
 
-  returnDailyConsumption() {
-
+  returnDailyConsumption(date, id) {
+  let dailyConsumption = this.hydrationData.find(consumption => {
+     return consumption.date === date && consumption.userID === id;
+    })
+     return dailyConsumption.numOunces;
   }
 
-  returnWeeklyConsumption() {
-
+  returnWeeklyConsumption(id) {
```

Comment:
I think this method is supposed to be more dynamic in that its calculations should be able to work for any specified week.  You could just create a variable that represent a week and it can be an array of seven consecutive days.  As long as you use that variable in the method it will be dynamic and work for whatever week is passed into the argument.  The parameters for the method would then be id, and week.  Slice would work to accomplish this but the start and end value would need to reflect the proper index points